base/db/R/query.prior.R

```diff
@@ -50,7 +51,7 @@ query.priors <- function(pft, trstr = NULL, con = NULL, ...){
           "`settings` object not found. Unable to connect to database."
         )
       }
-      params <- settings[[c("database", "bety")]]
+      params <- update[[c("database", "bety")]]
```

Comment:
This won't work because `update` will just be a logical scalar, not the entire settings object. I think the change to make here is to have `query.prior` give up if no params are passed in `...`. Specifically, lines 45-54 can be deleted and  replaced with

```
PEcAn.logger::logger.severe(
  "No connection (`con`) specified and no connection parameters given in `...`.",
  "Unable to connect to database.")
```

Reply:
@infotroph let's say we need to make this work. can simply assigning settings to our update variable in line 53 solve this?

Reply:
@moki1202 Yes, it would be possible to change the design to pass the whole settings object rather than subsets of it, but I think in this case, since it's only needed as the last of *three* possible ways of creating the connection, it's cleaner to remove it and let the function fail if `con` isn't provided.